src/orca-log-catcher/OrcaLogger.res

```diff
@@ -421,8 +424,28 @@ let make = (
     metadata := value
   }
 
+  let calculateAndUpdateCounterHook = eventName => {
+    let updatedCounter =
+      1 +
+      switch eventsCounter.contents->Js.Dict.get(eventName) {
+      | Some(num) => num
+      | None => 0
+      }
+    eventsCounter.contents->Js.Dict.set(eventName, updatedCounter)
+    Js.log((eventName, updatedCounter, "counter"))
+    updatedCounter
+  }
+
   let conditionalLogPush = (log: logFile) => {
-    if GlobalVars.enableLogging {
+    let maxLogsPushedPerEventName = 100
+    let eventName =
+      log.eventName->eventNameToStrMapper ++
+        switch log.eventName {
```

Comment:
Could you please add this value in a separate variable to improve code readability